### DISCUSSION TITLE
Share your custom overlays

### DISCUSSION THREAD

**Discussion by rusty-snake** [Helium314/SCEE#574] [Show and tell]
- Give new users better examples
- Create better overlays with the help of the community
- Prototype ideas for overlays integrated into StreetComplete/SCEE

_Moved to a [repository](https://github.com/rusty-snake/SCEE-custom-overlays) with proper linking support and full git history._

**Comment by Helium314** [comment]
Thanks, sharing overlays is a nice idea!
Both of my "normal" custom overlays are already on your list:
* access (in my case only for ways, and excluding `highway=platform`)
* maxspeed (coloring to also match `source:maxspeed`, and access is used for dash-filter instead of in element selection)

**Comment by goldfndr** [comment]
### tiger:reviewed

- Filtering details:
- -  `tiger:reviewed = no`
- - ways
- Coloring details:
- - *(empty)*
- - false
- - *(empty)*
